packages/vite-node/src/server.ts

```diff
@@ -428,7 +428,7 @@ export class ViteNodeServer {
     }
 
     const sourcemap = this.options.sourcemap ?? 'inline'
-    if (sourcemap === 'inline' && result && !id.includes('node_modules')) {
+    if (sourcemap === 'inline' && result) {
```

Comment:
I think similar change was proposed before:
- https://github.com/vitest-dev/vitest/pull/6041#discussion_r1668253799

> We don't inline source map because it's expensive to do for every file in node modules

Reply:
Yes, we can make it to opt-in if broken debugger and stacktrace for inlined deps are acceptable by default, but I'm leaning towards just aligning with Vite SSR.

Reply:
Yes, we had a discussion with the rolldown team not so long ago and decided to remove this check